### ISSUE TITLE
'documentation/source/requirements.txt' is missing some requirements

### ISSUE THREAD

**Issue by joshuacwnewton** [neuropoly/spinalcordtoolbox#3209]
The requirement.txt file contains 2 entries:

https://github.com/neuropoly/spinalcordtoolbox/blob/5bc7d0763cab8456fa99b58bbaee77411e04207e/documentation/source/requirements.txt#L1-L2

But the documentation [requires 4 packages](https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-Documentation#building-the-documentation):

```
pip install sphinx sphinx_rtd_theme sphinxcontrib.programoutput recommonmark
```

I usually end up ignoring the `requirements.txt` file because of this. This is a 5 second fix, though. :P

**Comment by 1nF0rmed** [comment]
Can I work on this?

**Comment by joshuacwnewton** [comment]
Certainly, @1nF0rmed! And, feel free to tag me as a reviewer when you are filling out the pull request template. This is a simple fix, so it will be quick to approve. :slightly_smiling_face:

**Comment by joshuacwnewton** [comment]
> Unless I'm missing something, shouldn't requirements.txt contain `sphinx` and `sphinx_rtd_theme`?

Ah, I was incorrect here. When I wrote this, I didn't realize we were using `extras_requires`:

https://github.com/neuropoly/spinalcordtoolbox/blob/d25e4ce12264ff75e5476fffaf7606342a459b87/setup.py#L42-L48

So, there is actually no need for `documentation/source/requirements.txt` anymore. It's simply a matter of removing the file and updating the internal dev docs.
